apis/v1beta1/collector_webhook_test.go

```diff
@@ -111,9 +111,7 @@ func TestCollectorDefaultingWebhook(t *testing.T) {
 			otelcol: OpenTelemetryCollector{},
 			expected: OpenTelemetryCollector{
 				ObjectMeta: metav1.ObjectMeta{
-					Labels: map[string]string{
-						"app.kubernetes.io/managed-by": "opentelemetry-operator",
-					},
+					Labels: map[string]string{},
```

Comment:
Hi @jaronoff97 ,
Just curious, after removing the filtering logic. For resources with label "app.kubernetes.io/managed-by": "Helm", could it be possible that both the helm chart and the OTel operator are managed at the same time resulting in a common modification of a resource (e.g., operator modifies the resource first, helm upgrade modifies it again)

Reply:
Different applications making changes to the same resource is a normal occurence in K8s, that by itself is not an issue. What we're fixing here is that the value of this label is a lie currently. The operator does not manage its own CRs in general, it only manages the ones it creates.

Reply:
yeah the issue today is that a collector or instrumentation created by helm will not be upgraded correctly by the operator because the label we filter for doesn't exist